hail/python/requirements.txt

```diff
@@ -1,13 +1,14 @@
+-c hailtop/pinned-requirements.txt
 -r hailtop/requirements.txt
+
 asyncinit>=0.2.4,<0.3
 avro>=1.10,<1.12
-bokeh==1.4.0
+bokeh>=3,<4
 decorator<5
 Deprecated>=1.2.10,<1.3
 dill>=0.3.1.1,<0.4
 frozenlist>=1.3.1,<2
 hurry.filesize>=0.9,<1
-Jinja2==3.0.3
 numpy<2
 pandas>=1.3.0,<2.1
```

Comment:
Does this requirement also need adjusting to `pandas>=2,<3`?

Reply:
Good catch thanks @jmarshall